llvmlite/tests/test_binding.py

```diff
@@ -3064,7 +3130,7 @@ def test_close(self):
         fpm = self.pm()
         fpm.close()
 
-    def test_run(self):
+    def test_run_O3(self):
```

Comment:
Changing case as suggestion below:

```suggestion
    def test_run_o3(self):
```

Also this could potentially use the `run_o_n()` function suggested below.